Automattic/sensei
# New Onboarding Experience feature

### Changes proposed in this Pull Request

* Add the New Onboarding Experience Feature (p6rkRX-4tb-p2)  to the trunk branch;

### Testing instructions

1. Verify if the new Onboarding Wizard works correctly;
2. Verify if Sensei Home works correctly;

### Notes

I'm creating this PR to allow us to track the plugin builds containing the changes of the New Onboarding Experience Project for Sensei LMS, so, this PR should NOT be merged yet. That's why it's still a draft. :)
